src/test/java/org/bahmni/mart/form2/translations/util/Form2TranslationsReaderTest.java

```diff
@@ -111,7 +118,6 @@ public void shouldReturnForm2TranslationsForGivenFormNameVersionAndLocale() thro
     @Test
     public void shouldReturnForm2TranslationsForGivenSpecialCharacterFormNameVersionAndLocaleWithNormalizedFilePath()
             throws IOException {
-
         String formNameWithSpecialCharacters = "2 Vitãls spéciælity_2.json";
         String normalizedFileName = "2_Vit_ls_sp_ci_lity_2.json";
         createFile(normalizedFileName);
```

Comment:
we are not going to use this kind of normalizedFormName (by replacing special char with underscore). we are using uuid instead of this. Need to modify the test case.

Reply:
We kept this to maintain backward compatibility